merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/MissionModelParser.java

```diff
@@ -411,17 +411,33 @@ private MapperRecord getExportMapper(final PackageElement missionModelElement, f
         ClassName.get((TypeElement) mapperType.asElement()));
   }
 
-  private List<ParameterValidationRecord> getExportValidations(final TypeElement exportTypeElement)
+  private List<ParameterValidationRecord> getExportValidations(
+      final TypeElement exportTypeElement,
+      final List<ParameterRecord> parameters)
+  throws InvalidMissionModelException
   {
     final var validations = new ArrayList<ParameterValidationRecord>();
+    final var parameterNames = parameters.stream().map(p -> p.name).collect(Collectors.toUnmodifiableSet());
 
     for (final var element : exportTypeElement.getEnclosedElements()) {
       if (element.getAnnotation(Export.Validation.class) == null) continue;
 
       final var name = element.getSimpleName().toString();
       final var message = element.getAnnotation(Export.Validation.class).value();
+      final var subjects = element.getAnnotation(Export.Validation.Subject.class) == null ?
+          new String[] { } :
+          element.getAnnotation(Export.Validation.Subject.class).value();
+
+      for (final var subject : subjects) {
+        if (!parameterNames.contains(subject)) {
+          throw new InvalidMissionModelException(
+              "Invalid validation subject \"%s\", only valid parameter names accepted (%s)"
+                  .formatted(subject, parameterNames.stream().map("\"%s\""::formatted).collect(Collectors.joining(", "))),
```

Comment:
This error message is a little noisy for a user. They can easily see the fields that do exist on the class; what we need to tell them are which validation subjects don't exist, and which validation method has this problem.

We're not actually telling them which validation method has the problem here, so they still have to do a little manual hunting to figure out where they need to make changes. It would be goAod to include this information in the error while suppressing the information they can already get easily.

Reply:
(I left the validation method name as "..." in my recommendation in a sibling comment. It doesn't seem to be in scope.)

Reply:
Error messages now look like:
```
BakeBananaBreadActivity.java:10: error: Validation subjects for validation "validateGlutenFreeTemperature" do not exist: "bad", "no_such"
```